src/components/Collections/Settings/Settings.vue

```diff
@@ -0,0 +1,428 @@
+        <div
+          v-if="isMobileView"
+          class="w-full flex justify-around space-x-2 mt-auto mb-10"
+        >
+          <!-- save button -->
+          <span v-tooltip="saveButtonTooltip" tabindex="0">
+            <!-- unsaved changes ping -->
+            <div
+              class="w-3 h-3 bg-primary animate-ping rounded-full absolute opacity-75"
+              v-if="hasUnsavedChanges"
+            ></div>
+            <icon-button
+              :buttonClass="saveButtonClass"
+              class="relative"
+              :titleConfig="saveButtonTitleConfig"
+              @click="saveChanges"
+              :isDisabled="!hasUnsavedChanges"
+              data-test="saveButton"
+            ></icon-button>
+          </span>
+        </div>
+      </div>
+      <!-- content region -->
+      <div :class="contentRegionClass" v-if="!isMobileView">
+        <div
+          v-for="(settingDetails, settingName) in currentSelectedTabDetails"
+          :key="settingName"
+          :class="settingItemStyleClass"
+        >
+          <div class="flex flex-col my-auto mr-4">
+            <p :class="settingTitleTextClass">{{ $t(settingDetails.title) }}</p>
+            <p :class="settingSubTitleTextClass" v-if="settingDetails.subTitle != null">
+              {{ $t(settingDetails.subTitle) || "" }}
+            </p>
+            <!-- badge to notify an admin setting -->
+            <simple-badge
+              v-if="settingDetails.isOrgSetting"
+              :text="$t('settings.badge.admin')"
+              :badgeClass="adminBadgeClass"
+            ></simple-badge>
+          </div>
+          <input
+            v-if="settingDetails.type == 'checkbox'"
+            type="checkbox"
+            :class="getInputElementClass(settingDetails.type)"
+            style="box-shadow: none"
+            :checked="settingDetails.value"
+            @change="updateCheckboxSetting($event.target.checked, settingDetails)"
+            data-test="setting-input"
+          />
+        </div>
+        <div class="w-full flex flex-col mt-10 bp-500:mt-0">
+          <!-- info for settings -->
+          <div
+            class="mt-12 sm:mt-8 md:mt-8 w-full p-1 bp-500:p-2 rounded-md border border-yellow-400 flex space-x-4 mb-4"
+            v-if="isInfoMessageVisible"
+            data-test="info-message"
+          >
+            <!-- icon -->
+            <div class="w-1/10 h-full flex my-auto">
+              <inline-svg
+                :src="getImageSource('exclamation-circle-solid.svg')"
+                class="lg:w-8 lg:h-8 bp-500:w-6 bp-500:h-6 h-4 w-4 text-yellow-600 fill-current my-auto transform rotate-180"
+              ></inline-svg>
+            </div>
+            <!-- text -->
+            <p
+              class="text-yellow-600 my-auto text-xs md:text-sm lg:text-base tracking-tighter font-bold"
+            >
+              {{ $t("settings.menu.info") }}
+            </p>
+          </div>
+        </div>
```

Comment:
I guess not
Or at least I'm not able to think of anything better

To avoid repeating code in HTML, components are used :p 
Toh iske andar bhi sub components bana denge toh ho jayega problem solve but I don't think it's worth it

Reply:
create issue